README.rst

```diff
@@ -40,31 +40,27 @@ Next step is to install assets (js, css etc.):
     plupload
     ...
 
-Now you can follow the standard invenio installation and development procedures, such as running `inveniomanage collect`.
+Now you can follow the standard Invenio installation and development procedures, such as running `inveniomanage collect`.
 
 ---------------------
 Demosite installation
 ---------------------
-As INSPIRE uses `Mixer <https://github.com/klen/mixer>`_ to load database fixtures, you should install the library:
```

Comment:
Mixer is in ``extras_require`` https://github.com/inveniosoftware/invenio/blob/pu/setup.py#L161 so still for INSPIRE we should tell people to install it.